### PR TITLE
django-haystack 

### PR DESCRIPTION
**Pull request by scopatz**
new noarch: python recipe

### CODE REVIEW

File: recipes/django-haystack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - django
```
Comment (synapticarbors) [inline recipes/django-haystack/meta.yaml]: The `setup.py` says django >= 1.11. Also, it looks like this uses `pkg_resources`, so you'll need `setuptools as a run time dependency.

**Comment by scopatz** [comment]
Thanks @synapticarbors! @conda-forge/staged-recipes, I think this is ready for reveiw

File: recipes/django-haystack/run_test.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# try to import haystack
```
Comment (ocefpaf) [inline recipes/django-haystack/run_test.py]: I don't know the first thing about `django` but maybe you can do something like https://github.com/conda-forge/django-test-plus-feedstock/blob/master/recipe/run_test.py?

I have also seen some 1-liners that add a Django context to make this work. I'll try to find an example for you.

Comment (scopatz) [reply recipes/django-haystack/run_test.py]: Good point @ocefpaf! it is worth a shot

**Pull request merged**